main.js

```diff
@@ -102,4 +102,15 @@ var draw = function() {
   time++;
 };
 
-draw();
\ No newline at end of file
+draw();d
+var rect = function(y) { 
+  （500，500，100，25)
```

Comment:
The function was correct before. You need to add `rect` again. I have added a commit to your pull request that does this. Please update your checked out branch and try it!